apps/condo/domains/property/constants/property.ts

```diff
@@ -1,3 +1,8 @@
+import { AddressMeta } from '@condo/domains/common/utils/addressApi/AddressMeta'
+
+// "д" - "дом" or "к" - "корпус"
+const validHouseTypes: AddressMeta['data']['house_type'][] = ['д', 'к', 'стр']
```

Comment:
Should I rename all constants in UPPER_CASE style?

Reply:
yes